packages/algob/src/internal/deployer.ts

```diff
@@ -275,13 +282,47 @@ export class DeployerDeployMode extends DeployerBasicMode implements AlgobDeploy
     return lsigInfo;
   }
 
+  /**
+   * Description: function to deploy stateful smart contracts
+   * @param approvalProgram filename which has approval program
+   * @param clearProgram filename which has clear program
+   * @param flags SSCDeploymentFlags
+   * @param payFlags Transaction Params
+   */
```

Comment:
Please add a link to the stateful sdk's here. [](https://developer.algorand.org/docs/features/asc1/stateful/sdks/#application-methods)